docs/backup-restore.md

```diff
@@ -175,3 +175,56 @@ Always make sure your backup strategy is tested and fits your needs.
 No one solution fits all, and we tried to cover the basic cases.
 For the latest updates, always feel free to ask on the BTCPay Server community channels.
 :::
+
+## How to migrate to a new server?
+
+The necessary steps always depend on your setup and infrastructure but as a rough guideline the following steps and considerations will get you through the process.
+
+:::warning
+If you have Lightning Network (LN) channels open on your internal node, you need to make sure to **not** restore any old channel state, otherwise your are at the risk of loosing funds. Therefore it is important to shut down BTCPay on the old server immediately after the backup has finished as outlined below.
```

Comment:
```suggestion
If you have Lightning Network (LN) channels open on your internal node, you need to make sure to **not** restore any old channel state, otherwise your are at the risk of losing funds.
Therefore it is important to shut down BTCPay Server on the old server immediately after the backup has finished as outlined below.
```